tests/baselines/reference/destructuringParameterDeclaration2.errors.txt

```diff
@@ -39,7 +39,6 @@ tests/cases/conformance/es6/destructuring/destructuringParameterDeclaration2.ts(
 !!! error TS2322: Type 'string' is not assignable to type 'number'.
                                 ~
 !!! error TS1005: ',' expected.
-!!! related TS1007 tests/cases/conformance/es6/destructuring/destructuringParameterDeclaration2.ts:7:4: The parser expected to find a ']' to match the '[' token here.

```

Comment:
Why'd we lose this one?

Reply:
Both `,` and `]` were expected, but `,` got there first, so the `]` error didn't get issued. With return value in hand, the code now skips issuing a `[]` related span on a `,` error.

Reply:
Can we create separate issue for reporting error like expected `,` or `]` instead.